### ISSUE TITLE
call_compartments: add check for binsize agreement between phasing track & cooler 

### ISSUE THREAD

**Issue by nservant** [open2c/cooltools#212]
Hi @nvictus ,

I successfully try to run `call-compartments` as follow ;

```
cooltools call-compartments -o test_comp SRR400264_1000000_norm.cool
chrom	start	end	E1	E2	E3
chr1	0	1000000			
chr1	1000000	2000000			
chr1	2000000	3000000			
chr1	3000000	4000000			
chr1	4000000	5000000	0.8903258045938355	0.4081314372994178	-0.13031754921806457
chr1	5000000	6000000	0.8977408617182799	-0.9590002539074324	1.711042047098714
chr1	6000000	7000000	0.2106623144257386	0.10316426992113314	-0.31927374642607137
chr1	7000000	8000000	-0.34651527590809367	-0.7671907595806922	0.2304382618662526
chr1	8000000	9000000	-0.5949924476220365	-0.06533560710647181	2.924361815527144
...
```

However, when I try to add the GC content reference tracks, the output looks strange.
```
cooltools genome binnify chrom_hg19.sizes 100000 > bins.txt
cooltools genome gc bins.txt hg19.fa > gc.txt
cooltools call-compartments --reference-track gc.txt  -o test_comp SRR400264_1000000_norm.cool
chrom	start	end	weight	GC	E1	E2	E3
chr1	0	1000000					
chr1	1000000	2000000					
chr1	2000000	3000000					
chr1	3000000	4000000					
chr1	4000000	5000000	0.20521617558629043				
chr1	5000000	6000000	0.15759258084022443				
chr1	6000000	7000000	0.16631149068711876				
chr1	7000000	8000000	0.1798014537315413				
chr1	8000000	9000000	0.17244550779615475				
chr1	9000000	10000000					
...
```

Actually, the eigen vectors are empty. I was thinking that the `--reference-tracks` was only used to sign the eigen values.
Did I miss something ?
Thank you

**Comment by gspracklin** [comment]
does call-compartments work if you calculate gc on 1mb size bins (to match the cooler size)?

**Comment by nservant** [comment]
arff yes, thank you.
I though I paid attention to that, but I lost a 0 in the battle 
Thanks

**Comment by gfudenberg** [comment]
this might be a good place to add a more informative error message!

**Comment by nservant** [comment]
Could you also explain me what it the output *.lam.txt file please ?
I did not find any details in the manual. Thanks !

**Comment by nvictus** [comment]
Those are the eigenvalues associated with the eigenvectors. They are closely related to "explained variance" in PCA. They can be useful for QC and also for reconstructing approximations to the obs/exp matrix using the eigenvectors. Higher-order eigenvectors will have lower eigenvalues as they contribute less to the original data.

We plan to have an example notebook diving more into the theory in the future: https://github.com/open2c/open2c_examples

**Comment by gfudenberg** [comment]
resolved with #320
